src/formattedcode/output_csv.py

```diff
@@ -82,7 +82,13 @@ def write_csv(results, output_file):
     w.writeheader()
 
     for r in rows:
-        w.writerow(r)
+        ordered_dict = r.copy()
+        if 'Resource' in ordered_dict:
```

Comment:
Why are you deleting these two rows?

Reply:
The empty rows comprise of two keys in the `OrderedDict`, which are `Resource` and `type`. The `Resource` key-value pair is present all the time whereas the `type` key-value is present half the times. So, after removing these two rows from the copy of the `OrderedDict`, we check if all the values are empty strings or not, and if any one of the value in the `OrderedDict` is non-empty, we write it.

Reply:
Long story short, we just wanted to have a check if the rows consisted of empty strings as values (`Resource` and `type` key-value pairs were not empty strings at times)

Reply:
@pombredanne I'd appreciate any better solution. :)